core/modules/file/config/file.settings.json

```diff
@@ -3,7 +3,7 @@
     "_module": "file",
     "max_filesize": "",
     "default_file_directory": "",
-    "default_allowed_extensions": "jpg jpeg gif png txt doc docx xls xlsx pdf ppt pptx pps ppsx odt ods odp mp3 mov mp4 m4a m4v mpeg avi ogg oga ogv weba webp webm",
+    "default_allowed_extensions": "jpg jpeg gif png txt doc docx xls xlsx pdf ppt pptx pps ppsx odt ods odp mp3 mov mp4 m4a m4v mpeg avi ogg oga ogv weba webp webm svg",
```

Comment:
Shouldn't that svg have been added to "file_default_allowed_extensions" instead? This might be the wrong spot, if it's about extensions for /file/add.

Reply:
Interesting question, but I cannot find where in core `file_default_allowed_extensions` is actually used. Could you point me to it?

Reply:
Hm... I think, you're right. It *is* default_allowed_extensions from file.settings.json. I always struggle to figure out, which one strikes.
Consider this as resolved (I have no permission to set this here):

Reply:
Created https://github.com/backdrop/backdrop-issues/issues/6298 to explore what thats all about.

Reply:
Good idea, this is really confusing. :+1: